Cargo.toml

```diff
@@ -42,7 +44,7 @@ version = "0.1"
 [features]
 default = ["std"]
 
-std = ["fnv/std"]
+std = ["fnv/std", "getrandom", "quickcheck"]
```

Comment:
I think we should have a separate feature for this. We don't want std to always enable this.

That feature can depend on std of course, if it is necessary.

Reply:
Done